packages/pyright-internal/src/analyzer/importResolver.ts

```diff
@@ -199,7 +199,7 @@ export class ImportResolver {
             );
 
             let success;
-            [success, current] = this._tryWalkUp(current);
+            [success, current] = this._(current);
```

Comment:
How does this work? What is `this._`?

Reply:
What the hell? Where did that happen?

Reply:
Uggh, that's what I get for using the github browser editor.